### PR TITLE
feat: duration ms in execution

### PR DESCRIPTION
**Pull request by exu**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: pkg/api/v1/testkube/model_execution_extended.go [hunk 1]
```diff
@@ -119,7 +120,10 @@ func (e *Execution) Start() {
 
 func (e *Execution) Stop() {
 	e.EndTime = time.Now()
-	e.Duration = e.CalculateDuration().String()
+	e.Duration = types.FormatDuration(e.CalculateDuration().String())
+	e.DurationMs = int32(e.CalculateDuration().Milliseconds())
+	fmt.Printf("%+v\n", e.DurationMs)
```
Comment (vsukhin) [inline pkg/api/v1/testkube/model_execution_extended.go:125]: is this for testing?

**Review by vsukhin — APPROVED** [review]
one comment and it's not compiled

File: pkg/api/v1/testkube/model_execution_extended.go (hunk 1 above)
Comment (exu) [reply pkg/api/v1/testkube/model_execution_extended.go:125]: 🤦🏻

Comment (exu) [reply pkg/api/v1/testkube/model_execution_extended.go]: removed thanks for catching this up! :)

**Comment by exu** [comment]
> 

fixed test

**Pull request merged**